lib/createWorkers.js

```diff
@@ -47,22 +49,81 @@ function createWorkers(handlers, config, options = {}) {
 
   let workerConnection;
   const workerChannels = [];
+
+  const TASK_COMPLETED = 'task.completed';
```

Comment:
I would move these const outside of the createWorkers function.

Reply:
ok